controllers/course.php

```diff
@@ -941,4 +967,30 @@ public static function nice_size_text($size, $precision = 1, $conversion_factor
         return $size;
     }
 
+    private function getEpisode($episodeId)
+    {
+        $series = OCSeminarSeries::getSeries($this->course_id);
+        if (empty($series)) {
+            return null;
+        }
+
+        $seriesId = $series[0]->series_id;
+        return \Opencast\Models\OCSeminarEpisodes::find([
+            $seriesId,
+            $episodeId
+        ]);
+    }
+
+    private function retractEpisode($episode)
+    {
+        $workflowClient = ApiWorkflowsClient::getInstance();
+        $result = $workflowClient->retract($episode->episode_id);
+        if (!$result) {
+            return false;
+        }
+
+        $episode->is_retracting = true;
+        $episode->store();
```

Comment:
@tgloeggl An dieser Stelle gibt es ein Problem. Ich schreibe zwar den neuen Wert in die DB. Nach kurzer Zeit wird der Wert aber wieder entfernt. Ich finde keine Stelle im Code, wo das passiert.

Reply:
Habe ich gelöst:
https://github.com/elan-ev/studip-opencast-plugin/commit/81007bcc5d8d3ea4db6c7113a676bc0c72f8a08b